### DISCUSSION TITLE
Could not create App Service certificate : domain.org with challenge : HTTP BadRequest : Could not add well known token to func07wcdzz.

### DISCUSSION THREAD

**Discussion by danielangcm** [rcl-ssl/docs#35] [Q&A] [answered]
`Could not create App Service certificate : domain.org with challenge : HTTP BadRequest : Could not add well known token to func07wcdzz. Site wwwroot failed to get Directory Info. Site wwwroot: failed to get Directory Info. {"Message":"'C:\\home\\site\\wwwroot\\.well-known\\acme-challenge\\' not found."}. SSL Azure App Service Install HTTP Validation Record - RCL V1 API
`
I was able to request for www.domain.org

But it failed for domain.org

**Comment by rcladmin** [comment]
There seems to be some issue with creating a record in the root of your website. The RCL app is trying to create a folder and add a file in the root of your app service folder but is being blocked. Do you have some security feature installed in your app service to block writing to the root of your website?

**Comment by danielangcm** [comment]
It’s working now. I selected the wrong site. Thanks

**Reply by rcladmin** [reply, marked as answer]
OK
